src/views/PurchaseOrder.vue

```diff
@@ -112,28 +143,35 @@ export default defineComponent({
           showToast(translate("Something went wrong, Please try again"));
         }
       },
-
       async parseFile(){
         await parseCsv(this.file).then(res => {
           this.content = res;
         })
       },
-      mapFields() {
+      review() {
+        if(!this.orderItemsList.length) this.mapFields();
+        this.store.dispatch('order/updatedOrderList', this.orderItemsList);
+        this.router.push({
+          name:'PurchaseOrderDetail'
+        })
+      },
+      mapFields(event) {
+        if(event && event.detail.value){
+          const mapping = JSON.parse(JSON.stringify(event.detail.value));
+          this.fields = mapping.fieldMapping;
+          this.mappingName = mapping.name;
+        }
         this.orderItemsList = this.content.map(item => {
```

Comment:
I think this should be done with review only

Reply:
Made the requested change.